src/Microsoft.AspNetCore.JsonPatch/Operations/OperationBase.cs

```diff
@@ -13,7 +14,12 @@ public OperationType OperationType
         {
             get
             {
-                return (OperationType)Enum.Parse(typeof(OperationType), op, true);
+                OperationType result;
+                if (!Enum.TryParse<OperationType>(op, ignoreCase: true, result: out result))
```

Comment:
Good question. I couldn't find any information about case sensitiveness in JsonPatch spec. Since  Json.net deserialization in general is case INsensitive, I believe this is fine? also it could be a breaking change to change this behavior now as we were ignoring case earlier too.